### ISSUE TITLE
Pull in VTT data before renamin glyphs to production names

### ISSUE THREAD

**Issue by madig** [googlefonts/gftools#700]
Is there a way to do this? Our designers use VTT on TTFs with designer names, but gfbuilder seems to rename the glyphs first before pulling in VTT data. This loses hinting. We can rename the glyphs to prodcution names in the VTT dumps, but that's wrong and I don't want to do it.

**Comment by m4rc1e** [comment]
Unfortunately there isn't a way to do this atm. We apply VTT hinting after the fonts have been generated.

**Comment by madig** [comment]
Yes. In our build system, we wait with post processing after everything else is done.
